apps/frontend/src/lib/components/Chat.svelte

```diff
@@ -86,14 +87,18 @@
 					updateMessages(chunk);
 			} catch (err) {
 				if (err instanceof Error) generation_error = err;
-				error(500, {
-					message: 'Error during generation'
-				});
```

Comment:
We still want this here as this is what triggers SvelteKit's and subsequently Sentry's error logging.